src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.person;
+
+/** 
+ * Store information for a contact
+ */
+
+public abstract class Contact {
+	
+    public String value;
```

Comment:
You have not passed in any values from the constructor. This way none of the class level members are getting assigned a value. It’s possible to push the constructor code to the parent class